truffle/src/main/ruby/core/rbconfig.rb

```diff
@@ -72,18 +72,14 @@ module RbConfig
       'LIBRUBY_A' => '',
       'LIBRUBYARG' => '',
       'prefix' => '',
-      'ruby_install_name' => 'jruby-truffle',
+      'ruby_install_name' => ruby_install_name,
       'RUBY_SO_NAME' => '$(RUBY_BASE_NAME)'
   }
   
   ruby_home = Truffle::Boot.ruby_home
 
   if ruby_home
-    bindir = if ruby_home.end_with?('/mxbuild/ruby-zip-extracted')
-               File.expand_path('../../bin', ruby_home)
-             else
-               "#{ruby_home}/bin"
-             end
+    bindir = "#{ruby_home}/bin"
```

Comment:
I think that's right - and then I've put a `truffleruby` in `bin`, which is a symlink to `bin/truffleruby` in our repository, and will be a symlink to the Ruby executable (which is in another place) in GraalVM and SVM. The link varies, but this way `bindir` and `RbConfig.ruby` don't have to.